ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -189,10 +208,17 @@ public void storeAggregatedStats(StatsSnapshot snapshot) throws SQLException {
         String containerIdKey = currContainerMapEntry.getKey();
         short containerId = Short.valueOf(containerIdKey.substring(2, containerIdKey.length() - 1));
         long currStorageUsage = currContainerMapEntry.getValue().getValue();
-        aggregatedaccountReportsDao.updateStorageUsage(accountId, containerId, currStorageUsage);
+        if (config.updateBatchSize >= 0) {
```

Comment:
We can probably use 'batchEnabled' boolean variable instead of config.updateBatchSize.

Reply:
updated.